admin/templates/common_auth/registration/invitation_email.html

```diff
@@ -0,0 +1 @@
+{% extends 'registration/password_reset_email.html' %}
```

Comment:
@Ghalko I would absolutely agree. The current override is arbitrary (but not totally generic). I need to find an OSF template to model after